cli/src/commands/deploy.ts

```diff
@@ -54,6 +54,11 @@ export default class Deploy extends Command {
                           Will create an app with a random name, for renaming use $ ${Command.cliBinName} rename`,
       exclusive: ['app-name'],
     }),
+    json: flags.boolean({
```

Comment:
[list command](https://github.com/reshufflehq/reshuffle/blob/96ab87bb4ff2c94b57ee79d35c9210812ba1a8a1/cli/src/commands/list.ts#L18) uses a `--format` flag - let's use the same format for consistency.

Reply:
Will do

Reply:
done